### PR TITLE
Tip12 JavaScript code snippets deprecated

### PR DESCRIPTION
**Pull request by janackeh**
- Updated content to match new content template
- Tested JavaScript tip several times, getting errors/tests fail, code snippet may need to be updated
- Errors I got : 2 test failed (0 passed) 1) A/B opt-out "before each" hook for "with cookie after visiting page": Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "quit()" is called; if returning a Promise, ensure it resolves. 

  2) A/B opt-out "after each" hook for "with cookie after visiting page": TypeError: Cannot read properties of undefined (reading 'quit')

### CODE REVIEW

**Comment by aquaomarine** [comment]
Adding a timeout on line 51 of the non-updated docs fixes the issues above and all test cases pass.

`beforeEach(async function() {
    this.timeout(60000)
    driver = await new Builder().forBrowser("chrome").build();
  });`

**Comment by janackeh** [comment]
Thank you @aquaomarine !!!!

File: frontend/docs/non-updated-tips/12-opt-out-of-ab-tests/javascript.md [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
+last_update:
+  date: 2023-03-09
 tags:
   - "ab tests"
   - "optimizely"
   - "opt-out"
 level: 2
 category: "testing"
+language: javascript
 ---
```
Comment (aquaomarine) [inline frontend/docs/non-updated-tips/12-opt-out-of-ab-tests/javascript.md:17]: Title

**Comment by janackeh** [comment]
title is there? 
Screenshot 2023-03-14 at 5 01 30 PM

**Comment by aquaomarine** [comment]
> title is there?
> Screenshot 2023-03-14 at 5 01 30 PM

Theirs a title in MD format # just above the intro. 
Screenshot 2023-03-14 at 7 05 32 PM

**Comment by janackeh** [comment]
Updated title

**Review by Roseywinter — APPROVED** [review]
Looks good to merge

**Comment by janackeh** [comment]
closing pull request - this file has been transferred to #193  - I created this one under unupdated tips instead of updated. #193  was created correctly under updated tips.

**Pull request closed without merging**